### DISCUSSION TITLE
Migration - different server

### DISCUSSION THREAD

**Discussion by Jurgens92** [kimai/kimai#2462] [Questions & Answers]
Hi All.

Ive got Kimai running on a local Hyperv Server, running on ubuntu 18(server)  and want to move this to our Vmware Esxi cloud server.

what would be the best way to move this server, 
lift and shit (and how) or backup and restore function? eg. sql backup and restore etc?

thank you

**Comment by kevinpapst** [comment]
mysql => mysqldump
reinstall Kimai on the new machine
dump => mysql
there is nothing in the filesystem expect the .env file and possible plugins (everything described in the backup documentation)
